extensions/namespace-lookup/src/main/java/io/druid/server/namespace/cache/NamespaceExtractionCacheManager.java

```diff
@@ -489,6 +515,49 @@ public void onFailure(Throwable t)
     }
   }
 
+  private static interface CallbackWaiter<T> extends FutureCallback<T>
```

Comment:
doesn't look like this interface is used anywhere else, we can probably remove it.